usr/share/rear/init/default/002_check_rear_recover_mode.sh

```diff
@@ -28,7 +30,7 @@ else
     # We are in the normal/original system:
     case "$WORKFLOW" in
         (recover|layoutonly|restoreonly|finalizeonly|mountonly)
-            Error "The workflow $WORKFLOW is only supported in the ReaR rescue/recovery system"
+            Error "The workflow $WORKFLOW is only supported in the ReaR rescue/recovery system, use --portable to disable this check."
```

Comment:
I think it could be dangerous to just tell the user to
```
use --portable to disable this check
```
because calling 'rear --portable recover'
on the original system is destructive
or do I misunderstand something here?

Reply:
Yes, that is destructive. But I cannot think of an indicator in portable mode that would tell us reliably if this is a rescue system or not. And I don't want to look at where `/` is mounted and try to guess something.

Happy to expand the message to say "at your own risk" or so :grin: